_sass/_sidebars.scss

```diff
@@ -70,6 +70,10 @@
     position: relative;
   }
 
+  ul {
```

Comment:
Without this, the sidebar has bullet points but I am not sure this is the best fix for that issue

Reply:
Hey,

**I'm not sure about this change**

I have two differents and suggested approaches to solve this problem and avoid next ones

1. Create a class named `has-bullets` and add in all elements that should **have** bullets
2. Create a class named `no-bullets` and add in all elements that should **not have** bullets

We can have more work know and fix it, avoiding future problems, or follow this approach and solve during the next problem occurrence.